.changeset/ten-bananas-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/por-address-list-adapter': minor
```

Comment:
Does this break the current EA? If so can we make this `major` ?

Reply:
This does not break the current EA